src/components/InternalLinkButton/InternalLinkButton.tsx

```diff
@@ -0,0 +1,88 @@
+import { styled } from '@mui/material';
+import Link from 'next/link';
+import React from 'react';
+import ArrowInternalLink from '../../stories/components/svg/ArrowInternalLink';
+
+interface Props {
+  label?: string;
+  icon?: boolean;
+  href: string;
+  className?: string;
+  iconFill?: string;
+  iconWith?: number;
+  iconHeight?: number;
+  iconClassName?: string;
+  background?: string;
+}
+
+const InternalLinkButton: React.FunctionComponent<Props> = ({
+  label,
+  icon,
+  href,
+  className,
+  iconFill = '#5B667E',
+  iconWith = 24,
+  iconHeight = 23,
+  background,
+  iconClassName,
+}) => (
+  <Link href={href ?? ''} passHref legacyBehavior>
```

Comment:
We should not use the `legacyBehavior` anymore, not in next >13, the `Link` component is rendered as a `a` tag so we don't need to add a nested `a` tab below. You can remove the `Container` and pass the logic to the styling of the `Link` itself

Reply:
Good to know , done